api/envoy/config/cluster/v3/cluster.proto

```diff
@@ -826,12 +826,8 @@ message Cluster {
   // from the LRS stream here.]
   core.v3.ConfigSource lrs_server = 42;
 
-  // If track_timeout_budgets is true, the :ref:`timeout budget histograms
-  // <config_cluster_manager_cluster_stats_timeout_budgets>` will be published for each
-  // request. These show what percentage of a request's per try and global timeout was used. A value
-  // of 0 would indicate that none of the timeout was used or that the timeout was infinite. A value
-  // of 100 would indicate that the request took the entirety of the timeout given to it.
-  bool track_timeout_budgets = 47;
```

Comment:
You cannot change an existing proto field. The existing field must remain, and a new field added, and optionally deprecate the existing field, but it must be supported according to the deprecation policy.

Reply:
not exactly sure, but I gave a try.
I am new to the community. please let me if the changes align the expectations.

Reply:
I am new to the community. Not exactly sure if the changes align the expectations, but I gave a try.